### ISSUE TITLE
Theme: WET - Menus fail to meet minimum colour contrast ratio requirements in mobile view.

### ISSUE THREAD

**Issue by michael-milette** [wet-boew/wet-boew#7119]
I've checked the mobile menus in the other themes and they all seem to be fine except for the WET theme.

As you can see, the menu text is kind of hard to see. The following screenshot was taken in Chrome 45 on the following page:

http://wet-boew.github.io/v4.0-ci/demos/index-en.html

![image](https://cloud.githubusercontent.com/assets/3808579/10172136/d27f29ae-66ab-11e5-80e3-67455d486121.png)

Mouse hover colour is fine.

Let me know if you require any additional information or have any questions.

Best regards,

Michael

**Comment by LaurentGoderre** [comment]
Duplicate of #6984 and waiting for #7023 to be merged to fix it
